homework1/parentheses/parentheses.c

```diff
@@ -0,0 +1,36 @@
+﻿#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+
+// Функция проверки баланса скобок.
+// На вход принимает указатель на строку со скобками и размер этой строки.
+// Возвращает true, если баланс корректный и false в противном случае.
+bool parenthesesBalance(char* inputString, size_t arraySize) {
+    int counter = 0;
+
+
+    for (size_t i = 0; i < arraySize; ++i) {
+        if (counter < 0)
+            return false;
+
+        if (inputString[i] == '(')
+            counter += 1;
+
+        else if (inputString[i] == ')')
+            counter -= 1;
+    }
+
+    return (counter == 0 ? true : false);
```

Comment:
```suggestion
    return counter == 0;
```
Иначе это тавтология какая-то